src/app.rs

```diff
@@ -166,6 +166,9 @@ pub fn app() -> App<'static, 'static> {
         .arg(flag("no-ignore-vcs"))
         .arg(flag("null").short("0"))
         .arg(flag("only-matching").short("o"))
+        .arg(flag("passthru").alias("passthrough")
+            .conflicts_with_all(&["only-matching", "replace"])
+            .overrides_with("count"))
```

Comment:
Hmm I kind of feel like `--passthru` should conflict with `--count`, no? They are somewhat mutually incompatible flags. I could be convinced though! Maybe explain this a bit more?

I agree with the other conflicts. Thanks for thinking about that!

Reply:
Hmmmm... I am conflicted. But you're right, we already have this problem today with other flags. I'm fine with the behavior in this PR, and adding `!self.is_present("count")` sounds good.